toqito/state_props/tests/test_is_abs_ppt.py

```diff
@@ -0,0 +1,69 @@
+"""Test cases for the `is_abs_ppt` function."""
+
+import numpy as np
+import pytest
+
+from toqito.state_props.is_abs_ppt import is_abs_ppt
+from toqito.states import bell, max_mixed
+
+
+@pytest.mark.parametrize(
+    "rho, dim",
+    [

```

Comment:
Can you test against the known examples of absolutely-PPT matrices? You can consult the "Examples" section of this paper for specific cases:
https://arxiv.org/pdf/quant-ph/0502170